Modified Graphs

adutfoy opened the pull request:
Introduced the use of the method setCorner()
Removed unseful Drawable.buildDefaultPalette()

mbaudin47 commented on python/doc/usecases/use_case_flood_model.rst:
```diff
@@ -412,10 +412,12 @@ This is a simpler model that leads to a simplified analysis.
         graph,
         figure_kw={"figsize": (6.0, 4.0)},
-        legend_kw={"bbox_to_anchor": (1.0, 1.0), "loc": "upper left"},
+        #legend_kw={"bbox_to_anchor": (1.0, 1.0), "loc": "upper left"},
```
I suggest to remove this line.

mbaudin47 reviewed (changes requested):
Now that the colors are automatically managed by the `Graph` class, there is, indeed, no need to use the `setColors()` in many scripts. Thank you for pointing this! Small changes will improve the PR.

mbaudin47 commented on python/doc/examples/functional_modeling/field_functions/plot_logistic_growth_model.py:
```diff
@@ -20,6 +20,7 @@
 from matplotlib import pylab as plt
 
 ot.Log.Show(ot.Log.NONE)
+ot.RandomGenerator.SetSeed(0)
```
This line is unnecessary.
```suggestion
```

jschueller replied:
no this is ok in examples because they are run in the same process

mbaudin47 commented on python/doc/usecases/use_case_flood_model.rst:
```diff
@@ -412,10 +412,12 @@ This is a simpler model that leads to a simplified analysis.
         graph,
         figure_kw={"figsize": (6.0, 4.0)},
-        legend_kw={"bbox_to_anchor": (1.0, 1.0), "loc": "upper left"},
+        #legend_kw={"bbox_to_anchor": (1.0, 1.0), "loc": "upper left"},
```
```suggestion
```

adutfoy replied:
yes!

mbaudin47 commented on python/doc/pyplots/DrawParallelCoordinates.py:
```diff
@@ -20,4 +20,7 @@
-View(cobweb, figure_kw={"figsize": (10, 6)}, legend_kw={"loc": "lower right"})
+cobweb.setLegendCorner([1.0, 0.0])
+cobweb.setLegendPosition('lower right')
+#View(cobweb, figure_kw={"figsize": (10, 6)}, legend_kw={"loc": "lower right"})
```
I suggest to remove this line.
```suggestion
```

adutfoy replied:
yes!

adutfoy commented:
@mbaudin47 : Michaël, I modified nothing in the file 
'python/doc/examples/calibration/least_squares_and_gaussian_calibration/plot_calibration_chaboche.py' 
but the test does not pass any more: do you understand what happens?
Thanks!
Anne

adutfoy commented:
I fixed some typos in the integration doc: 

- FilonQuadrature: wrong title
- SimplicialQuadrature: integration over an interval --> over a mesh
- CubaIntegration: optimization --> integration

I moved the class ExperimentIntegration from the DOE section to the Integration section in the User Manual.

jschueller commented:
needs a rebase

mbaudin47 reviewed (changes requested):
- The commits seems confusing to me: it seems that unnecessary commits were artificially introduced by Git. 
- I cannot access the HTML generated doc: is it available?

jschueller commented:
@adutfoy could you rebase this PR ?

jschueller commented on python/src/SimplicialCubature_doc.i.in:
```diff
@@ -24,14 +24,14 @@ Examples
 
 %feature("docstring") OT::SimplicialCubature::integrate
-"Evaluation of the integral of :math:`f` on an interval.
+"Evaluation of the integral of :math:`f` on an mesh.
```
on a mesh

jschueller commented:
moved to #2774
